search/intermediate_result.cpp

```diff
@@ -24,7 +24,6 @@ namespace search
 {
 
 /// All constants in meters.
```

Comment:
Now it is only one constant in meters. Then, may be write this comment on right side of declarations? Like this:
double const DIST_SAME_STREET = 5000.0; // in meters